### DISCUSSION TITLE
[Feature Request] Allow multiple columns sorting on Table

### DISCUSSION THREAD

**Discussion by CharriereThibaud** [heroui-inc/heroui#4749] [Feature requests]
### Is your feature request related to a problem? Please describe.

The parameter sortDescriptor of Table only take one sorting descriptor, so we can't implement multicolumn sorting in tables, even if we implement our proper sort function. The problem is not that we can't do multicolumn sorting with our function but we can't display sort arrow on multiple sorted columns.

### Describe the solution you'd like

It should be gread that sortDescriptor parameters allow to take a list to handle display of multicolumns sorting. Something like this:
```ts
export interface Sortable {
- sortDescriptor?: SortDescriptor,
+ sortDescriptor?: SortDescriptor[],
  onSortChange?: (descriptor: SortDescriptor) => any
}
```
and of course all logic linked to it.

That should permit to do this:
```tsx
const sortedColumns = [{column: "TITLE",direction: "ascending"}, {column: "NAME", direction: "descending"}] as sortDescriptor[]
<Table
        selectionMode="multiple"
        selectedKeys={selectedKeys}
        onSelectionChange={setSelectedKeys}
        aria-label="Articles"
        sortDescriptor={sortedColumns}
        onSortChange={(columnId) => sortTable(columnId)}
        key={tableKey}
        ...props
      >
        ...childs
</Table>
```
![image](https://github.com/nextui-org/nextui/assets/50869741/d3759cd2-6b82-4036-a81c-7d28bd4f9605)

### Describe alternatives you've considered

***

**Comment by sorcerb** [comment]
Any news?
